picard/coverart/image.py

```diff
@@ -25,24 +25,19 @@
 import shutil
 import sys
 import tempfile
-
 from hashlib import md5
-from PyQt5.QtCore import QUrl, QObject, QMutex
+
+from PyQt5.QtCore import QMutex, QObject, QUrl
+
 from picard import config, log
 from picard.coverart.utils import translate_caa_type
 from picard.script import ScriptParser
-from picard.util import (
-    decode_filename,
-    encode_filename,
-    replace_win32_incompat,
-    imageinfo
-)
+from picard.util import (decode_filename, encode_filename, imageinfo, replace_win32_incompat)
```

Comment:
Ok for the sort, but imho, multiline imports should be kept in the following form, one element per line, with extra comma at the end:

```python
from picard.util import (
    decode_filename,
    encode_filename,
    imageinfo,
    replace_win32_incompat,
)
```

Reasons:

- easier to sort in editors (ie. using `!sort` in vim)
- easier to insert/delete an element (keeping elements ordered, line deletion)
- easier to split when imported stuff is moved in another file
- patches easier to review (one change per line)
- shorter lines
- simpler expressions to search/replace